microsoft-365/security/mtp/overview-security-center.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Overview of the Microsoft 365 security center
-description: Describes the new home for monitoring and managing security across your Microsoft identities, data, devices, and apps.
+title: Overview - Microsoft 365 security center - Microsoft 365 security
```

Comment:
Saying Microsoft 365 security twice seems pretty redundant, is that what we are doing @DaniHalfin?

Reply:
@jdmartinez36 - you can remove the suffix here. There's already a suffix appended through build configuration ("Microsoft 365 security" is already configured as the suffix).